Chandan/chandan_practice.c

```diff
@@ -1,106 +1,199 @@
-#include <stdio.h>
-#include <time.h>
-#include <math.h>
-
-#define SECONDS_PER_MINUTE 60
-#define MINUTES_PER_HOUR 60
-#define HOURS_PER_DAY 24
-#define PROMPT_STRING "Type any key to show time, q to show time then quit: "
-
-#define TIME_TYPE unsigned long 
-
-char getChar() ;
-void updateTimes() ;
-void initTimes() ;
-void printPrompt() ;
-void printTime (TIME_TYPE timeInit) ;
-
-TIME_TYPE start, end;
+#include<stdio.h>
+#include<math.h>
+#include<stdlib.h>
+#include<string.h>
+#define MAX 50
+int top = -1, front = 0;
+int stack[MAX];
+void push(char);
+void pop();
+void addn(float,float, float);
 
 int hello_c()
 {
-        char command = 0;
-        initTimes();
-        delay();
-        updateTimes();
-        printTime(end-start);
-        return 0;
-}
+   float a = 14.5;
+   float b = 16;
+   float c = 17.5;
+   float *p, *q, *r;  
+   p = (int*)malloc(sizeof(int));
+   q = (int*)malloc(sizeof(int));
+   r = (int*)malloc(sizeof(int));
+   p = &a; 
+   q = &b;
+   r = &c;
+   int x = 3;
+   int y = 6;
+   int z = 5;
+   int i, HCF, max, step, LCM;
+   int j;
+   char s[MAX] = "radar", g;
+   float sum, prod, divs, mod, modu, cira, cirb;
+   float sub, diff, divi, u, area, cir, rec;
+   printf("\n The values are A = %f, B = %f, C=%f \n", *p,*q, *r );
+   printf(" \n The values of X = %d, Y = %d, Z= %d \n", x,y,z);
+   
+   void (*fun_ptr)(float, float, float) = &addn;
+   (*fun_ptr)(a, b, c);
+   
+   //printf("\n The sum of all the numbers");
+   //sum = *p + *q + *r;
+   //printf(" \n sum = %f \n", sum);
+   
+   printf("\n The difference of B and A ");
+   sub = *q - *p;
+   printf("\n difference = %f", sub);
+   
+   printf("\n The difference of C and B");
+   diff = *r - *q;
+   printf("\n difference = %f \n ", diff);
+   
+   printf(" \n The product of all numbers ");
+   prod = a * b * c;
+   printf("\n the product = %f \n", prod);
+   
+   printf("\n The division of B and A ");
+   divs = *q / *p;
+   printf(" \n The quotient = %f  \n", divs);
+   
+   printf("\n The division of C and B ");
+   divi = *r / *q;
+   printf("\n The quotient = %f \n", divi); 
+   
+   printf(" \n The even and Odd numbers of X, Y and Z :");
+   if( x%2 == 0)
+   printf ("\n X is an even number which is: %d ", x);
+   else
+   printf("\n X is an odd number which is : %d ", x);
+   
+   if( y%2 == 0)

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'y%2==0' is always true](https://app.codacy.com/manual/e-mock/CProgramming-Team6/pullRequest?prid=5597669)